src/std/env.ab

```diff
@@ -76,52 +89,59 @@ pub fun is_root(): Bool {
     return false
 }
 
+/// ?
 pub fun printf(format: Text, args: [Text] = [""]): Null {
     unsafe ${nameof args}=("{format}" "\$\{{nameof args}[@]}")$
     unsafe $printf "\$\{{nameof args}[@]}"$
 }
 
+/// ?
```

Comment:
escape special character for use in printf format.
I will give more explication